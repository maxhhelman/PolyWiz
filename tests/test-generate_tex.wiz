def int main()
{
  string[] body;
  string tex_doc;
  int[] ind;
  string intro;
  string outro;
  string fp;
  ind = [1];
  intro = "Edwards is a good prof";
  outro = "Hans is a chill TA";
  fp = "shrihan.png";

  body = [intro, fp, outro];

  tex_doc = tex_document(body, ind);

  printstr(tex_doc);

  return 0;
}