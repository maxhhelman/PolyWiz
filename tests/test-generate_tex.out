\documentclass{article}
\usepackage{graphicx}
\begin{document}
Edwards is a good prof\\
\begin{figure}[h]
\centering
\includegraphics[width=2.5in]{shrihan.png}
\label{fig_sim}
\end{figure}
Hans is a chill TA\\
\end{document}